src/aind_metadata_extractor/models/fip.py

```diff
@@ -0,0 +1,46 @@
+"""Data models of extracted information from FIP files."""
+
+from typing import Literal, Union, Optional, List
+from datetime import datetime
+from pathlib import Path
+from pydantic import BaseModel, Field
+
+
+class FiberData(BaseModel):
```

Comment:
Shouldnt this follow the same name as the acquisition? Fiber data seems too general no? FipDataModel perhaps?